frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card-body.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { LoadingInline } from '@console/internal/components/utils';
+
+export const DataConsumptionBody: React.FC<DataConsumptionBodyProps> = ({
```

Comment:
you dont need this component, same functionality is included in `DashboardCardBody`